[2025-10-15 23:08:15 +0700] [59937] [INFO] Starting gunicorn 23.0.0
[2025-10-15 23:08:15 +0700] [59937] [ERROR] Connection in use: ('127.0.0.1', 6000)
[2025-10-15 23:08:15 +0700] [59937] [ERROR] connection to ('127.0.0.1', 6000) failed: [Errno 48] Address already in use
[2025-10-15 23:08:16 +0700] [59937] [ERROR] Connection in use: ('127.0.0.1', 6000)
[2025-10-15 23:08:16 +0700] [59937] [ERROR] connection to ('127.0.0.1', 6000) failed: [Errno 48] Address already in use
[2025-10-15 23:08:17 +0700] [59937] [ERROR] Connection in use: ('127.0.0.1', 6000)
[2025-10-15 23:08:17 +0700] [59937] [ERROR] connection to ('127.0.0.1', 6000) failed: [Errno 48] Address already in use
[2025-10-15 23:08:18 +0700] [59937] [ERROR] Connection in use: ('127.0.0.1', 6000)
[2025-10-15 23:08:18 +0700] [59937] [ERROR] connection to ('127.0.0.1', 6000) failed: [Errno 48] Address already in use
[2025-10-15 23:08:19 +0700] [59937] [ERROR] Connection in use: ('127.0.0.1', 6000)
[2025-10-15 23:08:19 +0700] [59937] [ERROR] connection to ('127.0.0.1', 6000) failed: [Errno 48] Address already in use
[2025-10-15 23:08:20 +0700] [59937] [ERROR] Can't connect to ('127.0.0.1', 6000)
